openid-4-verifiable-presentations-1_0.md

```diff
@@ -1492,7 +1492,139 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
         </front>
 </reference>
 
-# Usage with Various Credential Formats {#alternative_credential_formats}
+<reference anchor="w3c.digital_credentials_api" target="https://wicg.github.io/digital-credentials/">
+        <front>
+          <title>Digital Credentials API</title>
+		  <author fullname="Marcos Caceres">
+            <organization>Apple Inc.</organization>
+          </author>
+          <author fullname="Sam Goto">
+            <organization>Google</organization>
+          </author>
+        </front>
+</reference>
+
+# OpenID4VP profile for the W3C Digital Credentials API
+
+This section defines a profile of OpenID4VP for use with the W3C Digital Credentials API [@!w3c.digital_credentials_api].
+
+The W3C Digital Credentials API defines a Web Platform API which allows web sites acting as Verifiers
+to request the presentation of Verifiable Credentials. The API itself does not define a credential exchange protocol
+but can be used with multiple protocols. The user agent working together with other layers of the platform/operating system and based on the permission of the End-User will send to the Wallet of the End-User's choice the request data along with the web origin of the Verifier.
+
+The design of this OpenID4VP profile utilizes the mechanisms of the W3C Digital Credentials API while also allowing to leverage advanced security features of OpenID4VP, if needed. It also defines the OpenID4VP request parameter that MAY be used with the W3C Digital Credentials API.
+
+The Digital Credentials API offers several advantages for implementers of both Verifiers and Wallets. 
+
+Firstly, the API serves as a privacy-preserving alternative to invoking Wallets via URLs, particularly custom schemes. The user agent and app platform will only invoke a Wallet if the user confirms the request based on contextual information about the credential request and the requestor (verifier). 
+
+Secondly, the user will always be returned to the initial context, typically a browser tab, when the request has been fulfilled (or aborted), which results in an improved user experience.
```

Comment:
This paragraph should cover cross/same-device: cross-device (QR Code), same-device (mobile browser/tab):
```suggestion
Secondly, the control flow will always be returned to the initial context, typically a browser tab, when the request has been fulfilled (or aborted), which results in an improved user experience.
```

Reply:
what is the `control flow`?

Reply:
I think 'control flow' is basically referring to "the application / browser tab that currently have focus / is in the foreground".

I agree that the suggested text is better than the current text, as the current text isn't great when considering cross-device flows.

Reply:
I don't understand the current sentence. can you please make a suggestion that is better than the current text but does not use a term control flow?

Reply:
How about 'the session with the user' instead?

```suggestion
Secondly, the session with the user will always continue in the initial context, typically a browser tab, when the request has been fulfilled (or aborted), which results in an improved user experience.```

Reply:
```suggestion
Secondly, the session with the user will always continue in the initial context, typically a browser tab, when the request has been fulfilled (or aborted), which results in an improved user experience.
```

thank you, Joseph!